File: 389-find-the-difference/find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int k = 0;
        for(auto i:s){
            k^= (i-'a');
        }
        for(auto i:t){
            k^=(i-'a');
        }
        return char(k+'a');
    }
};

File: 389-find-the-difference/find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int[] alphabets = new int[26];
        for (char c : s.toCharArray()) {
            alphabets[c - 'a']++;
        }
        for (char c : t.toCharArray()) {
            alphabets[c - 'a']--;
        }
        char c = 'a';
        for (int i = 0; i < 26; i++) {
            if (alphabets[i] == -1) {
                c = (char) (i + 'a');
            }
        }
        return c;
    }
}